frontend/src/components/Founder/Founders.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Typography from "@mui/material/Typography";
+import styles from "./Founders.module.scss";
+
+function Founders() {
+  return (
+    <>
+      <Typography variant="h2" color="primary">
+        Message from our founders
+      </Typography>
+      <div className={styles.mainContainer}>
```

Comment:
use object array and map it to render all the blocks

Reply:
const foundersData = [
  {
    imgSrc:
      "https://cdn.britannica.com/99/236599-050-1199AD2C/Mark-Zuckerberg-2019.jpg",
    name: "Mark Zingerburger",
    message: `"Founder of Facebook, Mark Zuckerberg is known for his innovative ideas and visionary leadership. He is also a pioneer in the field of technology and has a strong work ethic and dedication."`,
    className: styles.Box2,
  },
  {
    imgSrc:
      "https://cdn.britannica.com/99/236599-050-1199AD2C/Mark-Zuckerberg-2019.jpg",
    name: "Mark Zingerburger",
    message: `"Founder of Facebook, Mark Zuckerberg is known for his innovative ideas and visionary leadership. He is also a pioneer in the field of technology and has a strong work ethic and dedication."`,
    className: styles.Box1,
  },
  {
    imgSrc:
      "https://cdn.britannica.com/99/236599-050-1199AD2C/Mark-Zuckerberg-2019.jpg",
    name: "Mark Zingerburger",
    message: `"Founder of Facebook, Mark Zuckerberg is known for his innovative ideas and visionary leadership. He is also a pioneer in the field of technology and has a strong work ethic and dedication."`,
    className: styles.Box3,
  },
];